ghcide/ghcide.cabal

```diff
@@ -232,12 +232,7 @@ library
       -Wwarn=duplicate-exports -Wwarn=dodgy-exports
       -Wwarn=incomplete-patterns -Wwarn=overlapping-patterns
       -Wwarn=incomplete-record-updates
-
-  -- ambiguous-fields is only understood by GHC >= 9.2, so we only disable it
-  -- then. The above comment goes for here too -- this should be understood to
-  -- be temporary until we can remove these warnings.
-  if (impl(ghc >=9.2) && flag(pedantic))
-    ghc-options: -Wwarn=ambiguous-fields
+      -Wwarn=ambiguous-fields
```

Comment:
Shouldn't this still be guarded by the pedantic flag?

Reply:
It is - expand the code section above :)